langchain/client/runner_utils.py

```diff
@@ -382,12 +389,9 @@ async def arun_on_examples(
     """
     project_name = _get_project_name(project_name, llm_or_chain_factory, None)
     client_ = client or LangChainPlusClient()
-    client_.create_project(project_name, mode="eval")
+    client_.create_project(project_name)
 
     results: Dict[str, List[Any]] = {}
-    evaluation_handler = EvaluatorCallbackHandler(
```

Comment:
Bug fix - in order to make sure all the evaluators complete, we need to wait for all, not this single handler